src/TemplateProvider.php

```diff
@@ -137,15 +138,27 @@ protected static function curl_call($url, $connectTimeoutMs = 100, $timeoutMs =
 
         if ($response === false || curl_errno($ch)) {
             $message = 'Could not fetch toolbar from '.$url.': ' . curl_error($ch);
-            Log::log(new Exception($message, 500), Log::ERR);
+            self::logException(new Exception($message, 500));
             return '';
         }
 
         if ($responseInfo['http_code'] < 200 || $responseInfo['http_code'] > 299) {
             $message = 'Could not fetch toolbar, http_status "'.$responseInfo['http_code'].'" from "'.$url.'" ';
-            Log::log(new Exception($message, $responseInfo['http_code']), Log::ERR);
+            self::logException(new Exception($message, $responseInfo['http_code']));
```

Comment:
Again this rears its ugly head - this will be a behavioural change. These errors happen all the time in our websites, and `SS_Log` would silently log them to a file/Raygun then allow script execution to continue. The way we've implemented the core error handler in framework means that by registering an `error` with monolog you will halt execution of the website.

I still to disagree with the decision to do this, but I can appreciate the justifications for it.

The workaround for now would be to log these errors as info/warning/debug level so they aren't raised to the browser.

What do you think?

Reply:
Most recent reference: https://github.com/silverstripe/silverstripe-framework/issues/8044

I note that it's now been classified as a bug rather than expected behaviour

Reply:
In the meantime, we could log it as a warning? Does that pop into Raygun?

Reply:
It may be that our current behaviour for "error" should be reserved for "critical", "emergency", or "alert"